### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#157] [Esercizi - Bertini]
![Screenshot 2023-10-23 185049](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/8b3907aa-4a74-4abe-8080-995a178b005b)

**Comment by daromacarlo** [comment]
![Immagine 2023-10-26 165654](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/b69955b0-1bba-453a-b56f-97d490357d94)
![Immagine 2023-10-26 165736](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/a45e2558-d2f4-44fa-8be2-0c0bf4731b67)
![Immagine 2023-10-26 165757](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/a07bfe49-86f2-461a-8651-ab032ba88ccf)

**Comment by daromacarlo** [comment]
aggiornati punti 1 e 2.
![Immagine 2023-11-10 132326](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/021436f6-4fc1-48a1-af64-5703b9ecbaaf)
![Immagine 2023-11-10 132355](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/ca8d21d5-08bb-44b6-9bce-9ef29c1ec108)
![Immagine 2023-11-10 132414](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/b1fd8dde-cdd9-4962-91ea-7bbdc645d0c1)
![Immagine 2023-11-10 132432](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/dec386bb-d1bf-4310-92a6-8744cf473f65)
![Immagine 2023-11-10 132459](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/089d63a0-2278-41db-858f-102fe57f6142)

**Reply by niccolo-fato** [reply]
Nel secondo esercizio come ti esce 4/24?

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/97389762/03ee2e70-11ea-4b17-bdef-fa43398f168f)

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/user-attachments/assets/5cd4e511-53f6-4931-a923-ce920f343371)
